### PR TITLE
[IMPROVEMENT] Adding Arch Linux instructions and other minor fixes to COMPILATION.MD

### PR DESCRIPTION
**Pull request by ArchitBhonsle**
**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.
- [ ] **I have mentioned this change in the [changelog](https://github.com/CCExtractor/ccextractor/blob/master/docs/CHANGES.TXT).**

**My familiarity with the project is as follows (check one):**

- [x] I have never used CCExtractor.
- [ ] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---

`docs/COMPILATION.md` was missing the instructions to install the dependencies for Arch Linux. I also did some minor formatting fixes.

### CODE REVIEW

File: docs/COMPILATION.MD [hunk 1]
```diff
@@ -6,7 +6,7 @@ Downloads for precompiled binaries and source code can be found [on our website]
 
 Clone the latest repository from Github
 
-```
+```bash
 git clone https://github.com/CCExtractor/ccextractor.git
```
Comment (cfsmp3) [inline docs/COMPILATION.MD:10]: Why do we want that if we're already in a terminal? :-)

File: docs/COMPILATION.MD [hunk 2]
```diff
@@ -25,7 +25,13 @@ RHEL:
...
 yum install -y glew-devel glfw-devel cmake gcc libcurl-devel tesseract-devel leptonica-devel clang
 ```
-Rust 1.54 or above is also required.[Install Rust](https://www.rust-lang.org/tools/install). Check specific compilation methods below, on how to compile without rust
+
+Arch:
+```bash
+sudo paru -S glew glfw curl tesseract leptonica cmake gcc clang
+```
+
+Rust 1.54 or above is also required. [Install Rust](https://www.rust-lang.org/tools/install). Check specific compilation methods below, on how to compile without rust.
```
Comment (cfsmp3) [inline docs/COMPILATION.MD:34]: I believe we'll need a higher Rust version any day now :-)

Comment (ArchitBhonsle) [reply docs/COMPILATION.MD:34]: All the following code blocks also have the that so I just added it for consistency.

File: docs/COMPILATION.MD (hunk 1 above)
Comment (ArchitBhonsle) [reply docs/COMPILATION.MD:10]: All the following code blocks also have the that so I just added it for consistency.

**Comment by ArchitBhonsle** [comment]
This probably got lost in all the other notifs for you but I replied with reasons for the choices I made. @cfsmp3 could you take a look at those?

**Review by cfsmp3 — APPROVED** [review]

**Pull request merged**